### ISSUE TITLE
Currently possible to change cache setting for current file on-the-fly?

### ISSUE THREAD

**Issue by HubKing** [mpv-player/mpv#13020]
Using either the runtime command or scripting, is it currently possible to change the cache setting for the current file, and only for the current file to cache to the end of the file and not to remove already played parts?

The reason is that if I know I will watch to the end of the file, but I know I will pause a lot. If the video is on a spinning disk, the disk would spin-down and spin-up. Also, I would keep the already played part's cache in memory so that I could go back without spinning the disk up. But I don't want to change the cache setting for all files (could be a waste of memory and disk reading, if the file is huge and I am not going to watch the entire file); only when necessary.

**Comment by guidocella** [comment]
`set file-local-options/demuxer-max-bytes 1GiB; set file-local-options/demuxer-max-back-bytes 1GiB`

**Comment by HubKing** [comment]
I had opened the command line before, but I forgot the shortcut, so I tried searching Google, but I could not find it so I asked ChatGPT and it told me the shortcut was `:`, but that did not work. I asked Google Bard, and it also said `:` was the shortcut. After searching Google a lot more, I finally found that the shortcut is \`. I wonder why those AI's say `:` is the shortcut.

ChatGPT

The command was my third attempt, because with my previous trials, it seemed that it misunderstood that I wanted to actual system terminal.

![image](https://github.com/mpv-player/mpv/assets/10900489/92d0ab2d-09f2-4d22-92c3-03cf6809af19)

Google Bard

![image](https://github.com/mpv-player/mpv/assets/10900489/6b344db9-c237-45df-aeca-9087386c08ed)
